packages/events/defaultSubscribers/obtain.js

```diff
@@ -9,12 +9,14 @@ module.exports = {
   handlers: {
     "obtain:start": [
       function() {
+        if (this.quiet) return;
```

Comment:
Aha, good find!  (Same comment on the rest of these. :) )

Reply:
It would be nice to find a way to not have to write this guard every time. Maybe we could do something like create a base class that wraps all handler methods or something, any ideas?

Reply:
Huh, maybe?  I don't really have a good idea.

Reply:
That's an interesting approach @eggplantzzz!

Reply:
It's a bit tricky because it is not the case that you want to prevent *every* handler from executing when `quiet` is on. Maybe there is a way to designate them to be silent when `quiet` is on...

Reply:
Depending on how it's set up, there could be a base implementation that short circuits under certain default conditions that is overridable for those that want to control the short circuit logic